src/styles/blocks/caledar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  justify-content: center;
```

Comment:
```suggestion
```
problem with layout not fixed.
to fix just remove justify-content
![Screenshot from 2023-05-21 17-58-50](https://github.com/mate-academy/layout_calendar/assets/33029864/d39d1f9c-8993-41db-87de-7cbd449bec86)

also `start-day` modifier does not work